nordicsemi/dfu/dfu_faults.py

```diff
@@ -25,8 +25,12 @@ def __init__(self, fault_type: DFUFaultType, target_dfu_stage: DFUStage, delay_s
 
         :param fault_type: type of fault to generate
         :param target_dfu_stage: desired DFU stage during which the fault should be generated
-        :param delay_s: delay in seconds (starting from the target DFU stage) after which fault should be generated.
-                        None to generate fault on every call (permanent fault)
+        :param delay_s: delay in seconds (counted from entering the target DFU stage) after which fault should be
```

Comment:
After our discussion I realized that it is counted not even since given DFU stage start, but since first `call_fault()` in that DFU stage. Which may or may not be at the same time.